src/ui/Analytics/PendingSearchEvent.ts

```diff
@@ -125,7 +136,7 @@ export class PendingSearchEvent {
         if (this.sendToCloud) {
           this.endpoint.sendSearchEvents(this.searchEvents);
         }
-        const apiSearchEvents = _.map(this.searchEvents, (searchEvent: ISearchEvent) => {
+        const apiSearchEvents = map(this.searchEvents, (searchEvent: ISearchEvent) => {
```

Comment:
Don't need to use underscore's each/map anymore

Reply:
The `searchEvents` property is `protected`, so it's difficult to know how it's being manipulated outside of the project. I will keep the underscore method for now since it's safer.